mungegithub/mungers/submit-queue.go

```diff
@@ -577,7 +578,7 @@ func (sq *SubmitQueue) AddFlags(cmd *cobra.Command, config *github.Config) {
 		[]string{},
 		"Comma separated list of jobs in Jenkins to use for stability testing that needs only weak success")
 	cmd.Flags().StringSliceVar(&sq.RequiredStatusContexts, "required-contexts", []string{}, "Comma separate list of status contexts required for a PR to be considered ok to merge")
-	cmd.Flags().StringVar(&sq.RetestBody, "retest-body", retestBody, "message which, when posted to the PR, will cause ALL `required-retest-contexts` to be re-tested")
+	cmd.Flags().StringVar(&sq.RetestBody, "retest-body", newRetestBody, "message which, when posted to the PR, will cause ALL `required-retest-contexts` to be re-tested")
```

Comment:
Does this flag even work? It looks like we always use newRetestBody regardless of the flag's value.